app/views/patients/data_entry.html.erb

```diff
@@ -20,7 +20,8 @@
       <%= f.select :last_menstrual_period_weeks, options_for_select(weeks_options), label: t('patient.dashboard.weeks_along'), autocomplete: 'off' %>
     </div>
     <div class="col-sm-6 hide-label">
-      <%= f.select :last_menstrual_period_days, options_for_select(days_options), label: '.', autocomplete: 'off' %>
+      <%= f.select :last_menstrual_period_days, options_for_select(days_options, patient.last_menstrual_period_days ), autocomplete: 'off', skip_label: true, help: t('patient.dashboard.called_on', date: patient.initial_call_date.strftime("%m/%d/%Y")) %>
```

Comment:
This one uses the instance variable. kinda old-timey.

We can skip the help text here, since these patients are all brand new.

```suggestion
      <%= f.select :last_menstrual_period_days, options_for_select(days_options, @patient.last_menstrual_period_days ), autocomplete: 'off', skip_label: true %>
```